__tests__/app.test.js

```diff
@@ -35,24 +35,49 @@ describe("/api/topics", () => {
   });
 });
 
-
-describe('GET /api', () => {
-
+describe("GET /api", () => {
   it("GET /api/topics - should return an array of topic objects", () => {
     return request(app).get("/api").expect(200);
   });
 
-  it('includes all endpoints ', () => {
+  it("includes all endpoints ", () => {
     return request(app)
       .get("/api")
       .then((response) => {
-        console.log(response.body);
-         expect(response.body).toEqual(endPoints);
-       
-        
+        expect(response.body).toEqual(endPoints);
       });
   });
 });
 
+describe("GET /api/articles/:article_id", () => {
+  test("200: responds with an article object containing the correct properties", () => {
+    return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then((response) => {
+        console.log(response.body);
+        const { article } = response.body;
+        expect(article).toEqual(
+          expect.objectContaining({
+            author: expect.any(String),
+            title: expect.any(String),
+            article_id: expect.any(Number),
+            body: expect.any(String),
+            topic: expect.any(String),
+            created_at: expect.any(String),
+            votes: expect.any(Number),
+            article_img_url: expect.any(String),
+          })
+        );
```

Comment:
We need to be a lot more specific here. While we care about it also having all the necessary properties, we also care about what those properties look like specifically - we know that the article_id should be 1, we can look at the author, title etc.